### PR TITLE
[15.0][MIG] stock_picking_batch_extended_account_sale_type: Migration to version 15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT40156

Depends on:

- [x] https://github.com/OCA/stock-logistics-workflow/pull/1156

ping @carlosdauden @ernestotejeda

### CODE REVIEW

**Comment by rousseldenis** [comment]
/ocabot migration stock_picking_batch_extended_account_sale_type

File: stock_picking_batch_extended_account_sale_type/models/stock_batch_picking.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class StockBatchPicking(models.Model):
+    _inherit = "stock.picking.batch"
+
+    def _get_domain_picking_to_invoice(self):
+        domain = super(StockBatchPicking, self)._get_domain_picking_to_invoice()
```
Comment (rousseldenis) [inline stock_picking_batch_extended_account_sale_type/models/stock_batch_picking.py]:
```suggestion
        domain = super()._get_domain_picking_to_invoice()
```

**Review by rousseldenis — APPROVED** [review]
Code review. Minor change

**Comment by rousseldenis** [comment]
@sergio-teruel To help reviewers test with runboat, you can include your dependency in test-requirements.txt file (in a separate commit)

**Comment by sergio-teruel** [comment]
Hi @rousseldenis How can I do it?  Thanks

File: stock_picking_batch_extended_account_sale_type/models/stock_batch_picking.py (hunk 1 above)
Comment (sergio-teruel) [reply stock_picking_batch_extended_account_sale_type/models/stock_batch_picking.py]: Changes Done!!

**Comment by rousseldenis** [comment]
> Hi @rousseldenis How can I do it? Thanks

It follows pip requirements format.

You can check this one for example:

https://github.com/OCA/product-attribute/pull/1216/commits/14fada9a163b6ab385d4f65b9915c4e6a0b0211e

For instance, it is composed by :

`<odoo module package name> @ git+https://github.com/OCA/<github repository>/refs/pull/<PR number>/head#subdirectory=setup/<module name>`

**Comment by sergio-teruel** [comment]
@rousseldenis Thanks for the info!! great!! in next PR I will try this... :+1:

**Comment by sergio-teruel** [comment]
Rebase done!!

**Review by CarlosRoca13 — APPROVED** [review]
LGTM :smile:

**Comment by pedrobaeza** [comment]
I have created this wiki entry to reflect about how to do this:

https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference(s)-to-another-pull-request(s)

Please do any comment you think to improve it.

**Pull request merged**